### ISSUE TITLE
[lxqt-panel] High CPU usage starting with post-0.9.0 commit 181

### ISSUE THREAD

**Issue by jeremywh7** [lxde/lxqt#686]
I have had high CPU usage (~60% on all four cores) ever since updating to lxqt-panel-git-0.9.0.181.g0e9f65b-1-x86_64 [I'm using Arch].  If I revert to lxqt-panel-git-0.9.0.179.g5da4d08-1-x86_64, all is fine (running it now).  I retried going back and forth a few times to confirm, and it is consistent.  I also tried using the latest from today, lxqt-panel-git-0.9.0.191.g5e24a99-1-x86_64 which includes 18f5de9: "plugin-sensors: (re)added warning timer fequency initialization", to no avail.

Note that I'm only using the (post-0.9.0) -git packages for lxqt-panel and its dependencies (lxqt-panel-git, lxqt-globalkeys-git, liblxqt-git, libqtxdg-git, liblxqt-mount-git), so perhaps the issue is a dependency on another -git package?

These are commits 180 and 181, respectively:
>Commits on Jun 03, 2015
>@palinek  plugin-clock: content down-scaling & proper size adjustments after config change	a4a16c2
>@palinek  panelconfig: default background opacity to 100 & compositing note…fixes lxde/lxqt#514	0e9f65b

**Comment by palinek** [comment]
I don't see such usage ... so I'm not able to downtrace the reason.
@jeremywh7 please, try to identify the compoment which causes such CPU load by removin/adding indivicual plugins from/into panel.

**Comment by jeremywh7** [comment]
Ok, I went back to the latest commit 191 package to check this out.  After taking a minute to realize the '+' and '-' button in Configure Panel are for 'Add' and 'Remove' [and not up/down, despite seeing up/down arrows there too :-| ], I noticed something interesting with top running in the background.

The high CPU usage goes away whenever the panel is unhidden [so https://github.com/lxde/lxqt/issues/629 oddly benefits this issue for now :-) ] and comes back when it hides.  I then also found that removing the Date/Time Widget fixes it, and adding it back reinstates the issue, so the issue occurs when both autohide -and- Date/Time are used.  I guess something with the plugin-clock change in commit 180 is the cause?  Thanks!

**Comment by palinek** [comment]
@jeremywh7 please, confirm that lxde/lxqt-panel@fb7d256 solves this issue

**Comment by jeremywh7** [comment]
High CPU usage is resolved; awesome!  But I did notice something else now, and only something that I changed this morning when testing the original issue...

I selected the option to 'Show date below time on new line' and now the time and date are skewed partially off-view to the right.

Well, I've been messing w/ the various month/day  and position settings...and now its correct again...  :-/

Unless you know of a cause offhand, I think we are good here.  :-)  If I continue to have issues, like after a future reboot, I will open a new Issue.  Thanks again, @palinek!
